src/components/Documentation/RightPanel/index.tsx

```diff
@@ -34,6 +35,31 @@ const RightPanel: React.FC<IRightPanelProps> = ({
   const [currentHeadingSlug, setCurrentHeadingSlug] = useState<string | null>(
     null
   )
+  const generateHeadingId = (slug: string): string => `links-${slug}`
```

Comment:
This ID was being used in many places, so I created a helper function to ensure consistency of the ID string.